### DISCUSSION TITLE
Improved names for approach different (AD) settings

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#20926] [Q&A]
lightly discussed in #18905, feel it merits its own discussion before any PRs for it are made.

here is what ive come up with so far:

![names](https://user-images.githubusercontent.com/74463310/197781274-8f0300a9-1e58-4734-8f6e-227d17b4892b.png)

very much open to other naming suggestions, just feel its important to improve because the old ones were rather unintelligble

**Comment by bdach** [comment]
What the mod needs is a visual preview panel in settings of how the animation is going to look. Not name changes that won't really help much (because there is no good way you can name these options).

**Reply by mk56-spn** [reply]
I agree but still think the names could be a bit better even with this in mind
